packages/fxa-auth-server/lib/senders/emails/templates/newDeviceLogin/en.ftl

```diff
@@ -3,5 +3,11 @@
 newDeviceLogin-subject = New sign-in to { $clientName }
 # Variables:
 # $clientName (String) - A client the user hasn't signed into before (e.g. Firefox, Sync)
-newDeviceLogin-title = New sign-in to { $clientName }
+newDeviceLogin-title-2 = Your Firefox account was used to sign in
```

Comment:
```suggestion
newDeviceLogin-title-2 = Your { -product-firefox-account } was used to sign in
```

Better to call the brand placeable.

Reply:
Oh yeah, thank you!  I was going to ask about this but forgot.  I was not sure if in this context "Firefox account" was considered a brand placeable.